website/vercel.json

```diff
@@ -77,6 +77,10 @@
       "source": "/setup/cloudjiffy",
```

Comment:
Thanks @kuberaspeaking - for raising this PR. 

- The `vercel.json` changes are not needed. Please revert to the original.
- Could you please update the forked repo with the latest and then update the PR?